testsuite/features/secondary/trad_migrate_to_minion.feature

```diff
@@ -92,15 +92,15 @@ Feature: Migrate a traditional client into a Salt minion
     And I wait until I see "has been deleted" text
     Then "sle_client" should not be registered
 
-  @susemanager
+@skip_if_salt_bundle
```

Comment:
is this the right change here? just making sure

Reply:
Yes, we have @ on left margin on all our files, excepted a few exceptions like this one.

After this PR, there will be 4 "exceptions" left.

Reply:
```
@susemanager
```
are executed only if the server has SUSE Manager installed and will
not run if Uyuni is detected.

Inside the testsuite, the scenarios that are tagged with

```
@uyuni
```
are executed only if the server has Uyuni installed and will
not run if SUSE Manager is detected.

So would @uyuni be not the right tag here?

Reply:
the issue is that this and the scenario below are the same but with different filters. but instead of the filters being susemanager and uyuni, they're salt bundle or not salt bundle
but this is different in 4.2

Reply:
for completeness, here is the definition used by the bundle tag:
```
def use_salt_bundle
# Use venv-salt-minion in Uyuni, or SUMA Head and 4.3
  $product == 'Uyuni' || ['head', '4.3'].include?($product_version)
end
```

Reply:
And a screenshot of the bug in action:

![image](https://user-images.githubusercontent.com/1932575/187487259-37493956-d834-4b2d-ab47-4feaff8fb0af.png)

Reply:
Interesting. Then I guess the changes make sense.